Tools/ecl_ekf/batch_process_logdata_ekf.py

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 import argparse
-import os
+import os, glob
 
 """
-Runs process_logdata_ekf.py on all the files in the suplied directory with a .ulg extension
+Runs process_logdata_ekf.py on the .ulg files in the supplied directory. ulog files are skipped from the analysis, if a 
+ corresponding .pdf file already exists (unless the overwrite flag was set). 
 """
 
-parser = argparse.ArgumentParser(description='Analyse the estimator_status and ekf2_innovation message data for all .ulg files in the specified directory')
+parser = argparse.ArgumentParser(description='Analyse the estimator_status and ekf2_innovation message data for the'
+                                             ' .ulg files in the specified directory')
 parser.add_argument("directory_path")
+parser.add_argument('-ow', '--overwrite', action='store_true',
```

Comment:
The `-ow` is a short option and thus should be a single letter only.

Reply:
ok, I changed it. Thank you @bkueng.